### DISCUSSION TITLE
Fully automated CSV import

### DISCUSSION THREAD

**Discussion by tempest-1973** [jgraph/drawio#3752] [Q&A]
Hi, has anyone been successful at fully automating a csv import. I have 100s of diagrams to be created from data and would love to use the csv import that works great in drawio

**Comment by nicola-attico** [comment]
I'm trying the same and it doesn't seem to work

**Comment by marc-mclean1** [comment]
your best bet may be to look into that online CSV importer where you can specify the style and the csv data separately.  The mini-app takes these two args and generates a draw.io diagram.  That could be something to look into.
